rpxdock/app/pilot/helix_beta.py

```diff
@@ -49,11 +49,26 @@ def get_helix_args():
       "--helix_max_primary_angle", type=float, default=None,
       help='maximum rotation around axis per step. default based on min_isecond is usually the right choice'
    )
-   parser.add_argument("--cart_cell_width", type=float, default=10,
-                       help='cartesian resolution of the initial search stage, default 10')
-   parser.add_argument("--angle_cell_width", type=float, default=30,
-                       help='angular resolution of the initial search stage, default 30')
+   # parser.add_argument("--cart_cell_width", type=float, default=10,
+   #                     help='cartesian resolution of the initial search stage, default 10')
+   # parser.add_argument("--angle_cell_width", type=float, default=30,
+   #                     help='angular resolution of the initial search stage, default 30')
+
+   parser.add_argument("--tether_xform", type=str, nargs='*', default=[],
+                       help='two structs with configuration to sample around')
```

Comment:
be a little more descriptive please, from this help I have no idea what you are talking about (: